src/read_write/ply.rs

```diff
@@ -273,29 +274,57 @@ macro_rules! read_casted_property {
     };
 }
 
+macro_rules! push_reader {
```

Comment:
Is `create_and_return_reading_fn!` used exclusively by `read_casted_property!`, and is `read_casted_property!` used exclusively by `push_reader!`? If so, why shouldn't we combine them into one macro?

Reply:
We can combine `push_reader` and `read_casted_property`, but the other doesn't make too much sense, as `create_and_return_reading_fn` is used 10 times inside `read_casted_property`, which would blow up the code...